docs/user-guide/02-conventional-features.md

```diff
@@ -620,10 +620,13 @@ Like with the previous `vertexMain` example, mixed entry point setups also suppo
 
 However, because of certain systematic differences between entry point types, a uniform being _global_ or _local_ will have very important consequences on the underlying layout and behavior.
 
-For most all entry point types, D3D12 and Vulkan use one common root signature to define both global and local uniform parameters. For these cases, Slang maps uniforms to this common root signature. 
+For most all entry point types, D3D12 will use one common root signature to define both global and local uniform parameters. 
+Likewise, Vulkan descriptors will bind to a common pipeline layout. For both of these cases, Slang maps uniforms to the common root signature / pipeline layout. 
```

Comment:
I would like us to avoid overusing the term "bind" in the documentation because it is very ambiguous.

Here, I would say "Likewise in Vulkan, the binding locations of uniform parameters are typically defined by a common pipeline layout. For both targets, Slang allocates a binding location for uniform parameters that will constitute the common root signature or pipeline layout."